lib/HeroBonus.h

```diff
@@ -709,13 +709,13 @@ class DLL_LINKAGE CBonusSystemNode : public virtual IBonusBearer, public boost::
 	void propagateBonus(std::shared_ptr<Bonus> b);
 	void unpropagateBonus(std::shared_ptr<Bonus> b);
 	void removeBonus(const std::shared_ptr<Bonus>& b);
+	void removeBonuses(const CSelector &selector);
+	void removeBonusesRecursive(const CSelector &s);

```

Comment:
put space around `&`. Whole file has bad formatting, but new additions should follow guidelines.